### PR TITLE
(AGAIN) Kills default chemical color, adds syndicate pill sprite, nerfs xenobiology

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
1. All chemicals now have a unique color instead of C8A5DC. I even purged it from the blob strain because FUCK C8A5DC
2. Colorful reagent and quantum barber thing also have random colors on init.
3. Adds unused syndie pill sprite, can't be selected in chemmaster but it's there if you need it for stuffs.
4. Regenerative Jelly dyes your hair purple when it enters your bloodstream.
5. Also tweaked some existing reagent colors around

## Why It's Good For The Game
1. Helps with distinguishing between chemicals without needing science goggles, and breaks immersion less.
2. not very colorful if it's the same color all the time huh uhuh
3. for later adds by other people
4. Xenobio bad, purpling people good.
5. Same as 1.

## Changelog
:cl:
spriteadd: All chemicals now have a unique color instead of C8A5DC. I even purged it from the blob strain because FUCK C8A5DC
code: Colorful reagent and quantum barber thing also have random colors on init.
spriteadd: Adds unused syndie pill sprite, can't be selected in chemmaster but it's there if you need it for stuffs.
balance: Regenerative Jelly dyes your hair purple when it enters your bloodstream.
spriteadd: Also tweaked some existing reagent colors around, like uranium going from silver to dark green.
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
The reason why this nerfs xenobiology is actually secretely a conspiracy for a stealth buff, im telling you, HE WANTS TO BUFF XENOBIO.

HE WANTS TO ADD CATSLIMEPEOPLE.

**Comment by carlarctg** [comment]
> The reason why this nerfs xenobiology is actually secretely a conspiracy for a stealth buff, im telling you, HE WANTS TO BUFF XENOBIO.
> 
> HE WANTS TO ADD CATSLIMEPEOPLE.

i will not deny that cat slimes are in my ideas document

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -208,7 +208,7 @@
 
 /datum/reagent/water/holywater/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
 	if(iscultist(M))
-		to_chat(M, "<span class='userdanger'>A darkness begins to spread its unholy tendrils through your mind, purging the Old God's influence!</span>")
+		to_chat(M, "<span class='userdanger'>A vile holiness begins to spread its shining tendrils through your mind, purging the Justiciar's influence!</span>")
```
Comment (actioninja) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Isn't ratvar the justicar? `iscultist` now only refers to blood cult.

**Review by actioninja — CHANGES_REQUESTED** [review]
you still got a stray file but it doesn't seem to have changes so it's not really a big deal.

**Comment by Krysonism** [comment]
That is the exact same colour polypyrylium oligomers dyes your hair. Kind of dilutes the effect.

**Comment by carlarctg** [comment]
> That is the exact same colour polypyrylium oligomers dyes your hair. Kind of dilutes the effect.

not my fault regen jelly turned out to be also purple sadly

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: i think it used to be is_servant_of_ratvar but got changed? i'll fix it

**Review by actioninja — CHANGES_REQUESTED** [review]
I knew something was fucky with the no changes, the indicator was that you changed permissions on it, update the perms of tgui-next/bin/tgui
`git update-index --chmod=+x tgui-next/bin/tgui`

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: fixed

**Comment by actioninja** [comment]
Congrats, you finally fixed it.

**Comment by Kingtrin** [comment]
Look at these downvotes smh MY HEAD. The change doesn't remove regenerative jelly's regeneration effect you code-illiterate nerds.

**Pull request merged**